website/sidebars.json

```diff
@@ -83,16 +92,6 @@
           "querying/query-context"
         ]
       },
-      {
-        "type": "subcategory",
-        "label": "Druid SQL",
-        "ids": [
-          "querying/sql-syntax",
-          "querying/sql-data-types",
-          "querying/sql-metadata-tables",
-          "querying/sql-translation"
-        ]
-      },
       {
         "type": "subcategory",
         "label": "Druid SQL Functions",
```

Comment:
If we keep this subcategory, "Functions" should be lowercase. (other subcategories and page names use sentence case.)